source/backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs

```diff
@@ -206,14 +206,14 @@ public IActionResult GetAcquisitionFileOwners([FromRoute] long id)
         [HasPermission(Permissions.AcquisitionFileView)]
         [HasPermission(Permissions.ContactView)]
         [Produces("application/json")]
-        [ProducesResponseType(typeof(IEnumerable<PersonModel>), 200)]
+        [ProducesResponseType(typeof(IEnumerable<AcquisitionFileTeamModel>), 200)]
         [SwaggerOperation(Tags = new[] { "acquisitionfile" })]
         public IActionResult GetAcquisitionTeamMembers()
         {
             // TODO: This endpoint needs to be fixed to support ORGANIZATIONS as team members (which were introduced by PSP-6960)
```

Comment:
this todo appears to have been addressed, can it be removed?

Reply:
removed